### DISCUSSION TITLE
Release Schedule

### DISCUSSION THREAD

**Discussion by mohe2015** [lit/lit#2365] [Q&A] [answered]
Hi,

is there any release schedule that is public? In particular I would like to know when https://github.com/lit/lit/pull/2345 will be released.

Maybe there is also a possibility to use these in advance but since this is a monorepo I think all setups are comparably complicated and not a simple change of the version in package.json

Thanks for any help!

Kind regards
Moritz

**Comment by aomarks** [comment, marked as answer]
> is there any release schedule that is public?

We have no published release schedule. For the most part, our plan going forward is to do frequent releases soon after each change lands. We have been a little slow with this recently due to holiday vacations.

> In particular I would like to know when #2345 will be released.

This particular change was released yesterday as @lit-labs/ssr v2.0.0: https://github.com/lit/lit/releases/tag/%40lit-labs%2Fssr%402.0.0
